packages/gallery/tests/styleParams/shouldIndexDirectShareLinkInSEO.spec.js

```diff
@@ -0,0 +1,69 @@
+import { GALLERY_CONSTS } from 'pro-gallery-lib';
+import GalleryDriver from '../drivers/reactDriver';
+import { expect } from 'chai';
+import { itemsWithDirectShareLink } from '../drivers/mocks/items';
+import ItemView from '../../src/components/item/itemView';
+
+describe('styleParam - shouldIndexDirectShareLinkInSEO', () => {
+  let driver;
+  let sampleItemViewProps;
+  let sampleItem;
+
+  beforeEach(() => {
+    driver = new GalleryDriver();
+    // sample image with directShareLink
+    sampleItem = itemsWithDirectShareLink[0];
+    sampleItemViewProps = driver.props.itemView(sampleItem);
+  });
+  it('should use "rel" attribute for preventing seo indexing - itemClick = expand', async () => {
+    Object.assign(sampleItemViewProps, {
+      styleParams: {
+        itemClick: GALLERY_CONSTS.itemClick.EXPAND,
+        shouldIndexDirectShareLinkInSEO: false,
+      },
+    });
+    driver.mount(ItemView, sampleItemViewProps);
+    await driver.update();
+    const linkProps = driver.find.selector('a').props();
+    expect(linkProps).to.include({ rel: 'nofollow' });
+  });
+
+  it('should not use "rel" attribute for preventing seo indexing - itemClick = expand', async () => {
+    Object.assign(sampleItemViewProps, {
+      styleParams: {
+        itemClick: GALLERY_CONSTS.itemClick.EXPAND,
+        shouldIndexDirectShareLinkInSEO: true,
```

Comment:
The difference between the two tests is only the `shouldIndexDirectShareLinkInSEO`. Can you reuse the code?

Reply:
not sure how. I think it's easier to read it this way (or is it not? what do you think?)

Reply:
I created an example for you in [this link](https://github.com/wix/pro-gallery/blob/88c814998003be280cefbe530daaa1282711ac1b/packages/gallery/tests/styleParams/shouldIndexDirectShareLinkInSEO.spec.js#L26).
I think that the reader understands the difference between the tests much better this way. 
This is for you to decide, it's just my preference

Reply:
cool, seems cleaner. I think I'll go with your approach, thanks